### ISSUE TITLE
Implement onboarding screens

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot-ios#94]
The onboarding part should be shown only once after the first login.

- [ ] Use MvvM
- [ ] Use nibs instead of story boards, it makes collaboration easier
- [ ] Remove Romanian names from class name, variables etc
- [ ] Remove hard coded values, move them into configuration objects/plist/constants

Implement onboarding activity with 3 screens:

![page 1](https://user-images.githubusercontent.com/1358055/63537401-5529bc00-c51e-11e9-9948-7eb2f62640e9.png)

![page 2](https://user-images.githubusercontent.com/1358055/63537411-5b1f9d00-c51e-11e9-921f-3dd41a53e43e.png)

![page 3](https://user-images.githubusercontent.com/1358055/63537415-5d81f700-c51e-11e9-821d-d1436fe5da79.png)

Figma wireframes available here: https://www.figma.com/file/21x1ui3YZJrGnpZNVnEuuQ/DRAFT-MV-mobile?node-id=0%3A1

**Comment by GeartHub** [comment]
I can pick this one up

**Comment by aniri** [comment]
> I can pick this one up, would you like this to have it in mvvm or storyboards?

@GeartHub please use MvvM, thanks! :smile:

**Comment by CristiHabliuc** [comment]
@GeartHub please use MvvM and xibs for interface, instead of StoryBoards. Thanks!

**Comment by GeartHub** [comment]
Maybe a bit of a silly question, but I got the app to work. Now it wants me to authenticate, do I have any way to fake this so I can get past the authentication screen ?

**Comment by aniri** [comment]
@GeartHub please use the following test account: 0722222222 123456

**Comment by CristiHabliuc** [comment]
@GeartHub check out the instructions for contributing here: https://github.com/code4romania/monitorizare-vot#contributing 

- Fork it (https://github.com/code4romania/monitorizare-vot/fork)
- Create your feature branch (git checkout -b feature/fooBar)
- Commit your changes (git commit -am 'Add some fooBar')
- Push to the branch (git push origin feature/fooBar)
- Create a new Pull Request
